src/pages/Projects/PopUpDialog/index.jsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react'
+import { Button } from 'primereact/button'
+import { InputText } from 'primereact/inputtext'
+import { Calendar } from 'primereact/calendar'
+
+const PopUpMessage = (props) => {
+  const [data, setData] = React.useState({
+    ProjectName: '',
+    StartDate: '',
+    PiNumber: ''
+  })
+
+  const [editData, setEditData] = React.useState({
+    ProjectName: props.source.project_name,
+    PiNumber: props.source.iteration_number,
+    StartDate: ''
+  })
+
+  let date = new Date(props.source.start_date)
+  const yyyy = date.getFullYear()
+  let mm = date.getMonth() + 1
+  let dd = date.getDate()
+  if (dd < 10) {
+    dd = '0' + dd
+  }
+  if (mm < 10) {
+    mm = '0' + mm
+  }
+  const formattedDate = dd + '/' + mm + '/' + yyyy
+  console.log(Date(formattedDate))
+
+  const onChange = (key) => (e) => setData({ ...data, [key]: e.target.value })
+  const onChangeData = (key) => (e) =>
+    setEditData({ ...editData, [key]: e.target.value })
+
+  const createProject = async () => {
+    try {
+      const body = data
+
+      console.log(data)
+      await fetch(`${process.env.REACT_APP_API_URL}/AddingProject`, {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify(body)
+      })
```

Comment:
the difference between fetch and Axios 😄, we do have it in the project.
You don't need to change it, I'm just showcasing Axios
```suggestion
      await axios.post(`${process.env.REACT_APP_API_URL}/AddingProject`, body)
```